foo.js

```diff
@@ -0,0 +1,149 @@
+import Alt from './'
+import React from 'react'
+import Render, { connect } from './utils/Render'
```

Comment:
You shouldn't be able to import `context` like that - this is only working because of Babel's fallback for when you have a single default export.